### PR TITLE
Update and improve the Chinese translation

### PR DESCRIPTION
**Pull request by JeansouLiu**

### CODE REVIEW

**Comment by aleksilassila** [comment]
Hi, are you possibly the same person that contacted me via Discord about this? If that's not the case, I have two versions of updated Chinese locales.

**Comment by JeansouLiu** [comment]
> Hi, are you possibly the same person that contacted me via Discord about this? If that's not the case, I have two versions of updated Chinese locales.

No, I haven't contacted you via Discord.

**Comment by aleksilassila** [comment]
I'm having a bit hard time reaching the person that originally created the translations and also updated the Chinese messages recently, so I was thinking maybe you two could try to coordinate together and produce one updated messages file for me. I'll link to his changes. Since I don't know any Chinese, I don't really know which translation should I use.

**Comment by aleksilassila** [comment]
[Here's a link](https://cdn.discordapp.com/attachments/774545792777322506/777066522651787302/messages_zh_cn.properties) to his changes. 

Also,  I noticed that I have messed up one locale message, `error.PRIVATE_ISLAND` should be `\u00a77[\u00a74\u00a7lError\u00a77] \u00a7cYou can't check private island's info.` instead of `info.CLEAR_CONFIRM` in English. What would be the correct translation in Chinese?

File: src/messages_zh_cn.properties [hunk 1]
```diff
@@ -1,110 +1,159 @@
...
+error.NO_WORLDEDIT=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u672a\u68c0\u6d4b\u5230\u0020\u0057\u006f\u0072\u006c\u0064\u0045\u0064\u0069\u0074\u002e
+error.ERROR=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u5185\u90e8\u9519\u8bef\u002e\u0020\u8bf7\u8054\u7cfb\u7ba1\u7406\u5458\u002e
+error.COOLDOWN=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u521a\u521a\u906d\u5230\u4e86\u4f24\u5bb3\u002e\u0020\u4f20\u9001\u5c06\u5728\u0020 {0}\u79d2\u0020\u540e\u5f00\u59cb\u002e
+error.ISLAND_SAVE_ERROR=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u65e0\u6cd5\u4fdd\u5b58 {0}.schem.
+error.INSUFFICIENT_FUNDS=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u4f60\u6ca1\u6709\u8db3\u591f\u7684\u94b1\u002e
+error.ISLAND_UNFINISHED=\u00a7\u0037\u005b\u00a7\u0034\u00a7\u006c\u5bb6\u56ed\u5c9b\u00a7\u0037\u005d\u0020\u00a7\u0063\u60a8\u7684\u5bb6\u56ed\u5c9b\u4ecd\u5728\u751f\u6210\u002e
 error.PRIVATE_ISLAND=info.CLEAR_CONFIRM
```
Comment (aleksilassila) [inline src/messages_zh_cn.properties:27]: This line is supposed to say `error.PRIVATE_ISLAND=\u00a77[\u00a74\u00a7lError\u00a77] \u00a7cYou can't check private island's info.` in English

**Pull request merged**